pkg/apis/app/v1alpha1/kogitoapp_types.go

```diff
@@ -277,6 +277,8 @@ type Builds struct {
 	// Builds have been stopped from executing
 	// +listType=set
 	Cancelled []string `json:"cancelled,omitempty"`
+	// Flag to let operator know if the given build is from file, set by CLI when user create a kogito service from a Kogito resource, e.g. dmn/drl/bpmn standalone files
+	BuildFromAsset bool `json:"buildFromAsset,omitempty"`
```

Comment:
I think this flag should be part of the CR spec [KogitoAppBuildObject](https://github.com/kiegroup/kogito-cloud-operator/blob/0f38a87706e3f16665a56c28d71205a3516926d7/pkg/apis/app/v1alpha1/kogitoapp_types.go#L95), not in the CR [status](https://github.com/kiegroup/kogito-cloud-operator/blob/0f38a87706e3f16665a56c28d71205a3516926d7/pkg/apis/app/v1alpha1/kogitoapp_types.go#L226).

Reply:
It was initially there, @ricardozanini suggested remove it from there.

Reply:
I think we can find a way to stop it from appearing in the CRD, I'll take a look and see if we can do it by using some annotations. That's way we can completely avoid it from being exposed externally. Putting it in the `status` doesn't make much sense, and users still have access to the flag by setting a value in the `status`.

Reply:
@ricardozanini @spolti It seems it is not possible NOT to expose this flag, because the CLI is an external tool. If the flag is not exposed, CLI won't work.  On the other hand, if CLI can use the flag, users can also use this flag with `kubectl` or `oc` client. Actually, I think users are able to submit the files and trigger a Build manually without using the CLI anyway, although it is not documented. In this case, it makes no difference to put the flag in `status`.

Reply:
@xiezhang7 not sure if I understand your comment, what are you suggesting?

Reply:
I think we should put the flag in the `spec` instead of `status`. Because the flag is not supposed to be hidden, it has to be exposed externally so that the CLI can use it. But exposing it in `status` doesn't make sense according to this [definition by kubernetes](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#object-spec-and-status). So I think we should put it in the `spec` and make proper documentation that it is for the CLI.

Reply:
flag removed.